drivers/gpio/Kconfig.tca6424a

```diff
@@ -0,0 +1,21 @@
+# TCA6424A configuration options
+
+# Copyright (c) 2022 Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_COMPAT_TI_TCA6424A := ti,tca6424a
```

Comment:
Update this based on pattern in https://github.com/zephyrproject-rtos/zephyr/pull/48085

Reply:
Done.